### DISCUSSION TITLE
Keybinding for next/previous tab?

### DISCUSSION THREAD

**Discussion by jsjoeio** [LunarVim/LunarVim#3529] [Q&A] [answered]
Very new here so hello!

Trying to figure out what the keybinding/command is to go to the next/previous tab:

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/3806031/204067897-63740e72-9fee-4f84-a8b4-2309a8be4de9.png">

Here I have two open: _README_ and _sh_

Clicking each tab goes to them but wondering how to do it via a command and eventually a keybinding.

I found [this on Reddit](https://www.reddit.com/r/neovim/comments/v8wfsd/tabs_and_windows_splitting_in_lunarvim/) but when I run the command `:tabn` nothing happens.

**Comment by LostInTheLogs** [comment, marked as answer]
by default it's `leader b b/n`

**Reply by jsjoeio** [reply]
OH! They're Buffers, not tabs 🤦🏼 That's why I couldn't find it when I was looking. Thank you!
